lib/plaid/version.rb

```diff
@@ -1,3 +1,3 @@
 module Plaid
-  VERSION = '1.4.3'
+  VERSION = '1.4.5'
```

Comment:
Thanks for catching, my bad. I haven't dealt with versioning in a while :). I said bump the minor version in my commit message and then just bumped the patch.